pkg/controller/import-controller.go

```diff
@@ -526,6 +523,7 @@ func (r *ImportReconciler) createImporterPod(pvc *corev1.PersistentVolumeClaim)
 		scratchPvcName:    scratchPvcName,
 		vddkImageName:     vddkImageName,
 		priorityClassName: cc.GetPriorityClass(pvc),
+		selectedNodeName:  cc.GetSelectedNodeName(pvc),
```

Comment:
dont really need to add another value to the args since you have the pvc in the args you can do the get when creating the pod